zp-relayer/workers/sentTxWorker.ts

```diff
@@ -112,15 +109,24 @@ async function handleReverted(
     waitingJobIds.push(wj.id)
 
     const { txPayload, traceId } = wj.data
-    const transactions = [txPayload]
+    let reschedulePromise: Promise<any>
+    if (Array.isArray(txPayload)) {
```

Comment:
Let's make it more explicit with something like explicit payload type?

Reply:
Yeah, makes sense